pkg/vpcmodel/commonConnectivity.go

```diff
@@ -6,26 +6,40 @@ SPDX-License-Identifier: Apache-2.0
 
 package vpcmodel
 
-import "github.com/np-guard/models/pkg/connection"
+import (
+	"github.com/np-guard/models/pkg/connection"
+)
+
+// GeneralResponsiveConnectivityMap describes connectivity
+type GeneralResponsiveConnectivityMap map[VPCResourceIntf]map[VPCResourceIntf]*detailedConn
 
 type GeneralConnectivityMap map[VPCResourceIntf]map[VPCResourceIntf]*connection.Set
 
```

Comment:
update documentation here (for `GeneralResponsiveConnectivityMap` and `GeneralConnectivityMap`)

Reply:
done